### DISCUSSION TITLE
Driving mode using data when offline

### DISCUSSION THREAD

**Discussion by BusyBird15** [osmandapp/OsmAnd#18331] [Q&A] [answered]
Whenever I'm using OsmAnd for driving, it uses  around 6mb/minute of my cell data. This is a problem since I have a limited phone plan. However, it only uses the data in the driving profile and no other, any online plugins have been disabled, and my map source is set to offline maps. How do I stop it from using so much data?

**App version:** 4.5.8 - 4.5.10 (both versions had the issue)
**Operating system:** Android 10

**Comment by pebogufi** [comment]
I use osmand for many years, and i never had this problem. I suppose a wrong setting which loads data in background.
- check all profiles that there are no overlay or underlay active which might load data continuously when moving
- ...

**Reply by BusyBird15** [reply]
No, I don't have any underlay or overlay maps turned on.

**Comment by BusyBird15** [comment]
And here are my map settings for the driving profile:
<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/119532048/2911322d-1f45-4991-9dee-3d90da0d11cb"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/119532048/c4512307-6006-4814-ae6f-c2d8b070c889"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/119532048/8df97f87-c3fb-42a7-a799-c4a066c07fb8"  width="300" height="auto"/> </th>
 <th><img src="https://github.com/osmandapp/OsmAnd/assets/119532048/5ac3e1fe-6680-4704-899c-6fd4b4b429f8"  width="300" height="auto"/> </td>
  </tr>
</table>

**Edit:** I just tried using map rendering engine 1 (not OpenGL) and it did *not* use any mobile data. I did not close the discussion because I still want it not to use mobile data on v2 because v1 is sooo slow.

**Reply by DmitryAlexei** [reply, marked as answer]
@BusyBird15 Weather may use mobile data (weather plugin is disabled with rendering version 1)
try to download weather forecast for offline use. See https://osmand.net/docs/user/plugins/weather#offline-forecast

**Reply by BusyBird15** [reply]
Strange enough, though I have my local forecast downloaded, this was the issue. After switching back to the v2 renderer and disabling the weather plugin, it no longer used any cell data. Thanks for the help!
